services/utils/constants.js

```diff
@@ -0,0 +1,21 @@
+"use client";
```

Comment:
Is this required?

Reply:
NextJS v13 app dir onwards yes, rn no

Reply:
Oh ok, thanks 👍  I thin this file should move to config though, otherwise  `utils` becomes such a dumping ground